### PR TITLE
First iteration test suite mvp

### PR DESCRIPTION
**Pull request by JNadeu**
Done:
- Mark test with mvp pytest decorator
- Add and fix export csv feature test
- Fix some test rate 
- Fix some test workbench
- Fix some test device
- Fix some test tags

### CODE REVIEW

File: tests/test_device.py [hunk 1]
```diff
@@ -297,6 +297,8 @@ def test_sync_execute_register_no_hid_tag_not_linked(tag_id: str):
     """
     tag = Tag(id=tag_id)
     pc = d.Desktop(**conftest.file('pc-components.db')['device'], tags=OrderedSet([tag]))
+    from flask import g
```
Comment (slamora) [inline tests/test_device.py:300]: May import statement be moved to the head of the file?

Comment (JNadeu) [reply tests/test_device.py:300]: Yes, I understand that it is better to always import globally. I'll change it.

File: ereuse_devicehub/resources/device/models.pyi [hunk 2]
```diff
@@ -152,7 +152,7 @@ class Computer(DisplayMixin, Device):
         self.components = ...  # type: Set[Component]
         self.actions_parent = ...  # type: Set[e.Action]
         self.chassis = ...  # type: ComputerChassis
-        self.owner_address = ... # type: UUID
+        self.owner_id = ... # type: UUID
```
Comment (forste) [inline ereuse_devicehub/resources/device/models.pyi:155]: in my opinion, this pull rquest should only be about tests and not involve other files such as models, views, schemas, etc

Comment (JNadeu) [reply ereuse_devicehub/resources/device/models.pyi:155]: Okay, fine with me. But if it is a necessary change for a test to pass correctly, in these cases I understand that we can make changes in the models, views, etc... To fix it!

File: tests/test_device.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (JNadeu) [reply tests/test_device.py]: Done!

**Comment by JNadeu** [comment]
@slamora If you want to try again to execute: `pytest -m mvp --maxfail=5 tests/`
It should work, although there are some tests that are in xfail temporarily. At least I would have to finish all the tests with 0 failed.

**Comment by fedjo** [comment]
From a quick view the request looks ok.

No changes to models found! :-)

Guys @forste @emmdim what do you think?

**Pull request merged**